pkg/pillar/persistcache/README.md

```diff
@@ -0,0 +1,36 @@
+# Persisted Cache (persistcache)
+
+## Use-case
+
+This package implements a persistance cache. The main use-case is to store data on filesystem and
+provide access to it via API. Limitation of current implementation is that data should be represented
+as string. This was developed for configuration services feature in EVE. Since there is an option for 
+user to provide object to store via inline query, EVE has to store it even after reboot. 
+
+From library user perspective this library works as `map[string]string` with respective files created for each key in root folder.
+
+## API & Usage
+
+Main structure is `pesistCache` outside of `persistcache` package it could be created via `Load` function:
+it is loads values from cache if they are present or created folder if there was none.
```

Comment:
Hard to parse. Is there a type about should say "create folder"?

Reply:
Rephrased it